src/components/Profile/Components/MobileTableComponent.tsx

```diff
@@ -0,0 +1,154 @@
+'use client'
+import * as utils from '@/utils'
+import * as interfaces from '@/interfaces'
+import Modal from '@/components/Global/Modal'
+import { useCallback, useState } from 'react'
+import * as consts from '@/constants'
+import { useRouter } from 'next/navigation'
+
+export const MobileTableComponent = ({
+    key,
+    primaryText,
+    secondaryText,
+    tertiaryText,
+    quaternaryText,
+    type,
+    avatar,
+    dashboardItem,
+    address,
+}: interfaces.IProfileTableData) => {
+    const [modalVisible, setModalVisible] = useState(false)
+    const router = useRouter()
+    const handleSendToAddress = useCallback(
```

Comment:
Can this not be done with a <Link> @pawell24 ?

Reply:
Fixed